config/projects/taskcluster.yml

```
@@ -82,6 +82,14 @@ taskcluster:
       # note: workerConfig not specified here as configuration
       # is stored directly on manually provisioned worker
 
+    gw-ci-freebsd:
+      owner: taskcluster-notifications+workers@mozilla.com
+      emailOnError: true
+      imageset: generic-worker
+      cloud: static
+      lifecycle:
+        reregistrationTimeout: 2592000 # 30 days
```

```suggestion
      lifecycle:
        reregistrationTimeout: 2592000 # 30 days
      workerConfig:
        genericWorker:
          config:
            runTasksAsCurrentUser: true
```

Don't you need this set if we're calling this a `gw-ci-*` pool?

---

static pools don't fetch their config from worker manager 😬 

they probably should, but they don't 😬

---

Ah sorry I didn't realize this was static!